devolutions-gateway/src/streaming.rs

```diff
@@ -73,5 +78,13 @@ pub(crate) async fn stream_file(
         };
     });
 
-    Ok(upgrade_result)
+    return Ok(upgrade_result);
+
+    fn get_recording_file_path(recording_folder_path: camino::Utf8PathBuf) -> anyhow::Result<camino::Utf8PathBuf> {
+        let manifest = JrecManifest::read_from_file(&recording_folder_path.join("recording.json"))?;
+        // TODO: handle multiple files
+        let file = manifest.files.get(0).context("no files in manifest")?;
+
+        Ok(recording_folder_path.join(&file.file_name))
+    }
```

Comment:
issue: Maybe we want the last (current) file? You can use `.last()` for that. Also rename the function to `get_last_recording_file` to reflect that.

Reply:
updated